time_tracker_api/technologies/technologies_api.py

```diff
@@ -0,0 +1,83 @@
+from flask_restplus import Namespace, Resource, fields
+
+ns = Namespace('technologies', description='API for technologies used')
+
+# Technology Model
+technology = ns.model('Technology', {
+    'id': fields.String(
+        readOnly=True,
+        required=True,
+        title='Identifier',
+        description='The unique id of the technology'
+    ),
+    'tenant_id': fields.String(
+        required=True,
+        title='Tenant',
+        max_length=64,
+        description='The tenant this technology belongs to'
+    ),
+    'name': fields.String(
+        required=True,
+        title='Name',
+        max_length=50,
+        description='Name of the technology'
+    ),
+    'created_at': fields.Date(
+        title='Date',
+        description='Date of creation'
+    ),
+    'created_by': fields.String(
+        required=True,
+        title='Type',
+        max_length=30,
+        description='Id of user who first added this technology',
+    ),
```

Comment:
Also, let's move these kinds of fields that are generated after creation, i.e. created_at, created_by, tenant_id, etc., to a model called `technology_response`, this way you will not require them when the technology is being created but this has to be autogenerated.

Reply:
Those would be those ReadOnly fields